content/doc/book/pipeline/syntax.adoc

```diff
@@ -807,6 +807,9 @@ provide when triggering the Pipeline. The values for these user-specified
 parameters are made available to Pipeline steps via the `params` object,
 see the <<parameters-example>> for its specific usage.
 
+Each parameter has a _Name_ and some sort of _Value_, depending on the parameter type.
+These name-value pairs will be exported as environment variables when the build starts, allowing subsequent parts of the build configuration (such as build steps) to access those values, e.g. by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).
+
```

Comment:
```suggestion
Each parameter has a _Name_ and  _Value_, which depends on the parameter type.
These name-value pairs are exported as environment variables when the build starts, allowing subsequent parts of the build configuration, such as build steps, to access those values.
For example, this can be performed by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).

```
small syntax/wording suggestion, let me know if you have any questions!

Reply:
Thanks @kmartens27 for the suggestion.
The wording seems much clear and easy to understand now.